free-2.md

```diff
@@ -58,11 +107,32 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+This is somewhat unique becuase there is already syntax provided by copying the
+language idea from the CS5 creators. However, I think that there would be
+space for someone working on this project to think about additional debugging
+options and the ways in which students would interface with the language.
+Because this DSL is focused on teaching, the creator would be focusing on the
+user of the language, not the systems aspects of the project. Additionally,
+because none of the commands in HMMM are complex, there should not be a very
+complciated systems aspects of the language implementation.
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+I think that this project is not too ambitious, because the idea is already
+well thought out and it is clear what each instruction in HMMM does. There is
+then room to create more or less complicted debugging aspects of the project,
+which could scale the project to be an appropriate amount of work.
+
```

Comment:
I think this is definitely a valuable project but I'm unsure of what more you could add to the updated edition of HMMM; I feel like the vocabulary for the language is already simple enough, and to simplify it further might run the risk of making un-assembly-like. You could make quality of life improvements like how Hallie Seay worked on updating line numbers, but in general the scale of it all seems a bit smaller compared to your other ideas.